server-common/src/main/java/com/datastrato/gravitino/server/authorization/NameBindings.java

```diff
@@ -0,0 +1,19 @@
+/*
+ * Copyright 2024 Datastrato Pvt Ltd.
+ * This software is licensed under the Apache License version 2.
+ */
+package com.datastrato.gravitino.server.authorization;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+import javax.ws.rs.NameBinding;
+
+public class NameBindings {
+
+  @NameBinding
+  @Target({ElementType.TYPE, ElementType.METHOD})
+  @Retention(RetentionPolicy.RUNTIME)
+  public @interface AccessControlInterfaces {}
+}
```

Comment:
Why do you need this?

Reply:
Jersey filter supports name binding with annotation. 
The method with the name binding annotation will call the filter with the same annotation before execution of the method.